cmd/ndm_daemonset/controller/disk.go

```diff
@@ -17,8 +17,8 @@ limitations under the License.
 package controller
```

Comment:
Can this file be completely replaced by `blockdevice/blockdevice.go` ?

Reply:
Yes, this file can be removed. But not with `blockdevice/blockdevice.go`. It can be completely removed, as `DiskInfo` is no more used. Even if we want to create a `Disk` resource again at some point, we can generate it from the `BlockDevice` struct